README.md

```diff
@@ -639,4 +644,4 @@ Tide utilizes an [MIT license][mit-license].
 [sqs-region]: https://docs.aws.amazon.com/general/latest/gr/rande.html#sqs_region
 [tide-slack]: https://wordpress.slack.com/messages/C7TK8FBUJ/
 [wp-slack]: https://make.wordpress.org/chat/
-[mit-license]: https://github.com/xwp/go-tide/blob/master/LICENSE
\ No newline at end of file
+[mit-license]: https://github.com/xwp/go-tide/blob/master/LICENSE
```

Comment:
These lines are identical ❓

Reply:
Yeah, no clue why that's showing up as I didn't touch those links. Weird.